app/screens/post_options/index.test.js

```diff
@@ -9,6 +9,7 @@ import * as commonSelectors from 'mattermost-redux/selectors/entities/common';
 import * as teamSelectors from 'mattermost-redux/selectors/entities/teams';
 import * as deviceSelectors from 'app/selectors/device';
 import * as preferencesSelectors from 'mattermost-redux/selectors/entities/preferences';
+import * as postUtils from 'mattermost-redux/utils/post_utils';
```

Comment:
```suggestion
import {canEditPost} from 'mattermost-redux/utils/post_utils';
```

0/5 but I guess you don't need all the functions to be imported

Reply:
That's true. Mock importing the individual function was tricky to me though. Maybe it's just because automated testing is new to me😅. But we can mock the import as a whole as well which avoids the wildcard import.